src/possess/gotchi.rs

```diff
@@ -165,8 +165,8 @@ fn gotchi_serialize() -> Result<(), AttributeParseError> {
         CONFIG
             .possession_archetypes
             .iter()
-            .position(|x| x.name == "Adorpheus")
-            .unwrap(),
+            .position(|x| x.name == "Gempheus")
```

Comment:
Why was this changed from Adorpheus to Gempheus?

Reply:
Because Adorpheus was removed from the game because we weren't sure how to introduce her to the loot tables.

Reply:
Which is why the test was failing, it couldn't (for very valid reasons) find any gotchi by the name of Adorpheus.